test/test_jit.py

```diff
@@ -1746,6 +1746,9 @@ def equation_format(x, y):
         def sublist_format(x, y):
             return torch.einsum(x, [0], y, [1], [0, 1])
 
+        def optimize_kwarg(x, y):
+            return torch.einsum('i,j->ij', (x, y), optimize=[0, 1])
```

Comment:
For now, JIT script only supports List[int] which matches the signature in native_functions. To support List[Tuple[int, int]] we need to override the JIT function in C++ as was done for sublist format.